spec/features/user_spec.rb

```diff
@@ -452,6 +452,41 @@
         expect(page).to have_css(".govuk-notification-banner.govuk-notification-banner--success")
       end
     end
+
+    context "when reinviting a user after initial confirmation email has been sent" do
+      let!(:other_user) { FactoryBot.create(:user, name: "new user", organisation: user.organisation, email: "new_user@example.com", initial_confirmation_sent: true) }
```

Comment:
very minor refactor
we can remove 469 and add `confirmation_token: "abc"` in the factory
we can remove `FactoryBot.` as create is aliased to be called on FactoryBot in the spec config somewhere.
line 474 we can refactor to `user_path(other_user)` and we can remove the ! from the let here on 457 as the other_user will be created when we attempt to navigate to that path which is fine